src/components/site/Platform.tsx

```diff
@@ -21,15 +21,15 @@ export const PlatformsSyncMap: {
     icon: "/assets/social/telegram.svg",
     url: "https://t.me/{username}",
   },
-  twitter: {
-    name: "Twitter",
-    icon: "/assets/social/twitter.svg",
-    url: "https://twitter.com/{username}",
-  },
-  twitter_id: {
-    name: "Twitter",
-    icon: "/assets/social/twitter.svg",
-    url: "https://twitter.com/i/user/{username}",
+  x: {
```

Comment:
Have you considered the need to be compatible with the original link(twitter.com)?